workshops/portfolio/README.md

```diff
@@ -61,7 +61,7 @@ And now we're done creating a GitHub account!
 
 Now that we have a GitHub account:
 
-1. Open [jsbin.com][jsbin]
+1. Open [jsbin.com][jsbin] in a new window
```

Comment:
I don't think we need to, at least for the time being.

I was on a Google Hangout with a beginner yesterday and watched them while they went through the workshop. They didn't seem to have any trouble understanding this.